README.md

```diff
@@ -16,6 +16,13 @@ A Swift driver for [FaunaDB](https://fauna.com)
 * Xcode 8
 * Swift 3
 
+## Documentation
+
+Check out our driver specific [documentation](http://faunadb.github.io/faunadb-swift/).
```

Comment:
I'd make this a bit more specific:

"Check out the Swift-specific [reference documentation]...."